metricbeat/module/elasticsearch/_meta/env

```diff
@@ -1,2 +1,2 @@
-ELASTICSEARCH_HOST=elasticsearch
-ELASTICSEARCH_PORT=9200
+ES_HOST=elasticsearch
```

Comment:
Is there a specific reason you changed this? I kept it identical to the module name as I initially intended to do some auto generation here. Will merge it as is but curious if there were some issues with the longer version.

Reply:
yes, it seems we use ES_HOST across all beats here: https://github.com/elastic/beats/blob/master/libbeat/tests/system/beat/beat.py#L546